config.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+/* eslint no-process-env: 0 */
+
+const env = process.env.NODE_ENV || 'production';
+
+module.exports = {
+  env: env,
+  govukNotify: {
+    notifyApiKey: process.env.NOTIFY_KEY,
+    caseworkerEmail: process.env.CASEWORKER_EMAIL,
+    userConfirmationTemplateId: process.env.USER_CONFIRMATION_TEMPLATE_ID,
+    businessConfirmationTemplateId: process.env.BUSINESS_CONFIRMATION_TEMPLATE_ID
+  },
+  redis: {
+    port: process.env.REDIS_PORT || '6379',
+    host: process.env.REDIS_HOST || '127.0.0.1'
+  },
+  upload: {
+    maxFileSizeInBytes: 25 * 1024 * 1024, // 25MiB in bytes
+    hostname: process.env.FILE_VAULT_URL,
+    allowedMimeTypes: [
+      'image/jpeg',
+      'image/jpg',
+      'image/png',
+      'application/pdf',
+      'application/msword',
+      'application/vnd.openxmlformats-officedocument.wordprocessingml.document',
+      'application/vnd.ms-excel',
+      'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet'
```

Comment:
Do we have any confirmation on this list yet?

Reply:
`a PNG, PDF, JPEG or Microsoft Word or Excel file`
the requirements are a bit loose, I included `JPG` as well and raised it with BA to confirm,
For Microsoft Word Excel the extensions included are - doc, docx, xls, xlsx

Reply:
Sure, we can refine from here when we have confirmation